internal/test/framework/framework.go

```diff
@@ -28,6 +31,9 @@ const (
 
 	// awsUsername is the default windows username on AWS
 	awsUsername = "Administrator"
+	// remoteKubeletLogPath is the directory where all the log files related to kubelet are generated on the remote Windows
+	// Machine
```

Comment:
This is the directory where we are dropping all the logs not just kubelet, so please reflect that here.

Reply:
Good Point. Will change it.